arch/s390/dfltcc_common.h

```diff
@@ -3,6 +3,72 @@
 
 #include "zutil.h"
 
+/*
+   Runtime Tuning parameters.
+*/
+
+/*
+  control usage of DFLTCC
+  
+  env variable: DFLTCC
+  0 or unset: use DFLTCC algorithm (default)
+  1: disable DFLTCC
+ */
+extern int env_dfltcc_disabled;
+
+/*
+  control usage of DFLTCC_CMPR 
+  set this to 1 if you need reproducible builds 
+
+  env variable: SOURCE_DATE_EPOCH
+  0 or unset: use DFLTCC_CMPR
+  1: disable DFLTCC_CMPR
+ */
+extern int env_dfltcc_source_date_epoch;
+
+/*
+  disable DFLTCC for specific compression levels
+
+  env variable: DFLTCC_LEVEL_MASK
+  valid range: 0-10 / 0x0-0xA
```

Comment:
It's a bitmask, so the valid range is [0, (1<<10)).

Reply:
actually its a [shift for a mask](https://github.com/zlib-ng/zlib-ng/blob/develop/arch/s390/dfltcc_deflate.c#L41).
I see three possibilities to fix this:

1. just fix the comment
2. rename env var to DFLTCC_LEVEL_MASK_SHIFT
3. change it to allow a 10 bit mask

Reply:
make totally sense. I'll fix the comment and also add a check for a valid mask when reading the env variable.